### PR TITLE
GEOMESA-2198 Checking for iterators on classpath during createSchema

### PR DESCRIPTION
**Pull request by elahrvivaz**
Signed-off-by: Emilio Lahr-Vivaz <elahrvivaz@ccri.com>

### CODE REVIEW

File: geomesa-web/geomesa-web-data/pom.xml [hunk 1]
```diff
@@ -44,7 +44,7 @@
             <dependency>
                 <groupId>org.apache.commons</groupId>
                 <artifactId>commons-vfs2</artifactId>
-                <version>2.0</version>
+                <version>2.1</version>
```
Comment (jnh5y) [inline geomesa-web/geomesa-web-data/pom.xml]: Should we externalize the a commons-vfs2.version?

Comment (elahrvivaz) [reply geomesa-web/geomesa-web-data/pom.xml]: we should probably just get rid of the spark endpoint tbh. i think that's the only reason this is here... it was causing unit test failures in this module with the older version.

**Comment by jnh5y** [comment]
Let's nuke it!

File: geomesa-web/geomesa-web-data/pom.xml (hunk 1 above)
Comment (jnh5y) [reply geomesa-web/geomesa-web-data/pom.xml]: Let's nuke it!

Comment (elahrvivaz) [reply geomesa-web/geomesa-web-data/pom.xml]: opened a ticket here: https://geomesa.atlassian.net/browse/GEOMESA-2210

**Pull request merged**